woocommerce/woocommerce-android
# Issue/10574 edit budget screen UI

Part of: #10574


### Description

This PR adds the following UI changes to Blaze campaign budget screen: 
- Adds the slider and texts to edit the budget
- Adds bottom sheet with forecast info
- Adds bottom sheet to edit campaign duration

The UI is responsive in general and the value are kept at the view model level, but this PR doesn't include the following: 
- Updating the forecasting data
- Return the selected budget values to preview screen

Those tasks will be added in subsequent PRs

### Testing instructions

1. Lauch Blaze campaign creation flow
2. Click on the budget row
3. Interact with the budget screen sliders and see how the numbers are updated
4. Open both bottom sheets and check the UI matches Figma designs.

For reference: 

![Screenshot 2024-01-23 at 17 26 35](https://github.com/woocommerce/woocommerce-android/assets/2663464/52f0507a-93c1-48c7-aaf9-c36867f63c63)

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/2663464/f0bc82f4-2e25-4d91-9df4-ae100c831b27
